library/src/main/java/com/google/android/exoplayer/drm/StreamingDrmSessionManager.java

```diff
@@ -386,6 +391,14 @@ private void onKeyResponse(Object response) {
     try {
       mediaDrm.provideKeyResponse(sessionId, (byte[]) response);
       state = STATE_OPENED_WITH_KEYS;
+      if (eventHandler != null && eventListener != null) {
```

Comment:
Depending on your license setup, this might be called many times during playback of a single video. For example if keys are valid for X minutes and fresh keys are loaded every Y minutes, where Y < X. So I don't think onDrmSessionReady is a great name. Maybe the listener method should be called onKeysLoaded, which accurately reflects what just happened?